src/ir/effects.h

```diff
@@ -684,10 +685,11 @@ class EffectAnalyzer {
     ReadsMemory = 1 << 6,
     WritesMemory = 1 << 7,
     ImplicitTrap = 1 << 8,
-    IsAtomic = 1 << 9,
-    Throws = 1 << 10,
-    DanglingPop = 1 << 11,
-    Any = (1 << 12) - 1
+    TrapsNeverHappen = 1 << 9,
+    IsAtomic = 1 << 10,
+    Throws = 1 << 11,
+    DanglingPop = 1 << 12,
```

Comment:
Why not add it at the end (or right before "Any")? that would reduce the risk of confusion, in theory. I don't see a downside?

Reply:
Done!